WebRoot/messages/ZmMsg.properties

```diff
@@ -4138,11 +4138,11 @@ recurWeeklyEveryWeekday = Every {0,date,EEEE}
 # {2} is marker to indicate new line for input, replaced with "" on output
 recurWeeklyEveryNumWeeksDate = Every {0,number,integer} week(s) on {2}{1,list,date,EEEE}
 recurMonthlyEveryNumMonthsDate = Day {0,number,integer} of every {1,number,integer} month(s)
-recurMonthlyEveryNumMonthsNumDay = The {0,choice,-1#last|1#first|2#second|3#third|4#fourth} {1,date,EEEE} of every {2,number,integer} month(s)
-recurMonthlyEveryNumMonthsWeekDays = The {0,choice,-1#last|1#first|2#second|3#third|4#fourth} {1,choice,-1#day|0#weekend day|1#weekday} of every {2,number,integer} month(s)
+recurMonthlyEveryNumMonthsNumDay = The {0,choice,-1#Last|1#First|2#Second|3#Third|4#Fourth} {1,date,EEEE} of every {2,number,integer} month(s)
```

Comment:
suggestion, should we capitalize these string via css and keep them all lowercase here?